src/Repository/LinkBlockRepository.php

```diff
@@ -423,54 +426,6 @@ private function getHookMaxPosition(int $idHook, int $idShop): ?int
         return null !== $maxPosition ? $maxPosition + 1 : 0;
     }
 
-    /**
-     * @param int $linkBlockId
-     * @param array $shopIds
-     *
-     * @throws DatabaseException
-     */
-    private function updateShopAssociation(int $linkBlockId, array $shopIds): void
-    {
```

Comment:
it's nice to get rid of this redundant logic, hopefully, this will be done in the core, the doctrine way 👍